From: Roger Shimizu <rosh@debian.org>
Date: Sun, 18 Mar 2018 23:12:26 +0900
Subject: Remove apparmor local path from setup.py

apparmor local files were just removed, so change setup.py accordingly.
---
 setup.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/setup.py b/setup.py
index 37452ba..cf098c1 100644
--- a/setup.py
+++ b/setup.py
@@ -83,10 +83,6 @@ if distro != 'Ubuntu':
                 'apparmor/torbrowser.Browser.firefox',
                 'apparmor/torbrowser.Browser.plugin-container',
                 'apparmor/torbrowser.Tor.tor']),
-            ('/etc/apparmor.d/local/', [
-                'apparmor/local/torbrowser.Browser.firefox',
-                'apparmor/local/torbrowser.Browser.plugin-container',
-                'apparmor/local/torbrowser.Tor.tor']),
             ('/etc/apparmor.d/tunables/', ['apparmor/tunables/torbrowser'])
         ]
 
